force_control_based_on_tip_position_controller:
  type: open_manipulator_controllers/ForceControlBasedOnTipPositionController
  joint_names:
    - joint1
    - joint2
    - joint3
    - joint4

  root_link: world
  tip_link: end_effector_link
